published/Standup/action_inputs.json

```diff
@@ -45,7 +15,7 @@
     "exportId" : "Text",
     "name" : "Text"
   },
-  "question" : "Any blockers at the moment?",
+  "question" : "What is blocking your work at the moment? Type \"none\" if you aren't blocked.",
```

Comment:
"none" reads as a grammatically incorrect response to that question. "nothing"?

Reply:
Done.